packages/logger/src/Logger.ts

```diff
@@ -205,7 +206,7 @@ class Logger extends Utility implements ClassThatLogs {
    * @returns {Logger}
    */
   public createChild(options: ConstructorOptions = {}): Logger {
-    return cloneDeep(this).setOptions(options);
+    return new Logger(merge({}, this.initOptions, options));
```

Comment:
Given that in the `constructor`, `options` already defaults to `{}` (when no options are passed), is the first `{}` in this `merge` needed?

Reply:
`merge` mutate the first argument, so `initOptions` of the parent will be mixed with the child's options. If we create a second child from the parent after that, it will have both parent and first-child options.

Reply:
Okay, thanks for explaining it. I'm not very familiar with `lodash` so I assumed it worked differently. 

Let's go with your implementation!